File: problem_forty.js
// The sum of the streams 2
var inp = 8
function Myfunction(inp){
    var count = 0 
    for (var i = 0; i < inp; i++){
        var mult = 1
        for (var j = 0; j<i; j++){
            var add = j* mult
            mult+= add
        }
        var div = i/mult
        count += div
    }
    return count

}
console.log(Myfunction(inp))

File: problem_forty.py
# The sum of the streams 2
inp = int(input("Enter the number:"))
def Myfunction(inp):
    count= 0
    for i in range(inp):
        add =1
        for  j in range(i):
            kk = j*add
            add += kk 
        nn =i/add
        count+=nn
    return count        
print(Myfunction(inp))    